lib/algorithms/hard-filter-feature-min-platform.js

```diff
@@ -5,7 +5,7 @@
  */
 
 /*
- * Copyright (c) 2019, Joyent, Inc.
+ * Copyright 2019 Joyent, Inc.
```

Comment:
2020 right?

Reply:
Not when I wrote this :) - updated now to 2020.